commitizen/providers.py

```diff
@@ -166,6 +167,87 @@ class NpmProvider(JsonProvider):
     filename = "package.json"
 
 
+class Npm2Provider(VersionProvider):
```

Comment:
As mentioned in the issue, I'd suggest we update the `NpmProvider` directly, but I'm not a JavaScript users. Would love to see how @woile thinks.

Reply:
I think so as well, if `package-lock.json` and or `npm-shrinkwrap.json` exists they should be updated.

Reply:
Makes sense. I'll go back and update the existing NpmProvider  and take out the Npm2Provider.

Reply:
Should I change the PR from feat to fix then?